packages/editor/src/components/editor-container/editor-container.ts

```diff
@@ -97,23 +90,23 @@ export class EditorContainer extends LitElement {
     const placeholderRoot = html`
       <default-page-block
         .mouseRoot=${this as HTMLElement}
-        .store=${this.store}
+        .space=${this.space}
         .model=${this.placeholderModel}
       ></default-page-block>
     `;
 
     const pageContainer = html`
       <default-page-block
         .mouseRoot=${this as HTMLElement}
-        .store=${this.store}
+        .space=${this.space}
         .model=${this.model}
       ></default-page-block>
     `;
 
     const edgelessContainer = html`
       <edgeless-page-block
         .mouseRoot=${this as HTMLElement}
-        .store=${this.store}
+        .space=${this.space}
```

Comment:
I think passing down `space` here is fine, just saying that in the future, it may require other spaces to be used together with this one (for some possible cross page operations). But since we'll have properties like `space.store` and wrap spaces into pages, it shouldn't be a big deal.